com.unity.multiplayer.mlapi/Editor/MLAPIProfilerModule.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using MLAPI.Profiling;
+using Unity.Profiling;
+using UnityEditor;
+using UnityEngine;
+
+namespace MLAPI
+{
+    [InitializeOnLoad]
+    public static class MLAPIProfilerModule
+    {
+#if UNITY_2020_2_OR_NEWER && ENABLE_PROFILER
+        const string RPCModuleName = "MLAPI RPCs";
+        const string OperationModuleName = "MLAPI Operations";
+        const string MessageModuleName = "MLAPI Messages";
+
+        /// <summary>
+        /// This needs to be in synced with the internal dynamic module structure to provide our own counters
+        /// </summary>
+        [Serializable]
+        public class MLAPIProfilerCounter
```

Comment:
Nit, these can be private I think. In fact I think the whole class should probably be internal since it doesn't expose any public APIs

Reply:
Done